### ISSUE TITLE
fail with installation of typegen: Can't install it

### ISSUE THREAD

**Issue by matrixise** [anttiviljami/openapi-client-axios#21]
node@9db768f22148:/usr/src/app/ntp-dashboard$ yarn --version
1.22.4
node@9db768f22148:/usr/src/app/ntp-dashboard$ node --version
v13.12.0

```
node@9db768f22148:/usr/src/app/ntp-dashboard$ yarn add openapi-client-axios
yarn add v1.22.4
[1/4] Resolving packages...
[2/4] Fetching packages...
info fsevents@1.2.12: The platform "linux" is incompatible with this module.
info "fsevents@1.2.12" is an optional dependency and failed compatibility check. Excluding it from installation.
info fsevents@2.1.2: The platform "linux" is incompatible with this module.
info "fsevents@2.1.2" is an optional dependency and failed compatibility check. Excluding it from installation.
[3/4] Linking dependencies...
warning " > sass-loader@8.0.2" has unmet peer dependency "webpack@^4.36.0 || ^5.0.0".
warning " > vuetify-loader@1.4.3" has unmet peer dependency "webpack@^4.0.0".
[4/4] Building fresh packages...
[-/4] ⢀ waiting...
[2/4] ⢀ ejs
[-/4] ⢀ waiting...
error /usr/src/app/ntp-dashboard/node_modules/openapi-client-axios: Command failed.
Exit code: 2
Command: npm run typegen:install
Arguments: 
Directory: /usr/src/app/ntp-dashboard/node_modules/openapi-client-axios
Output:
npm WARN lifecycle The node binary used for scripts is /tmp/yarn--1586597873701-0.30829400471264035/node but npm is using /usr/local/bin/node itself. Use the `--scripts-prepend-node-path` option to include the path for the node binary npm was executed with.

> openapi-client-axios@3.3.5 typegen:install /usr/src/app/ntp-dashboard/node_modules/openapi-client-axios
> cd typegen && npm install

sh: 1: cd: can't cd to typegen
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! openapi-client-axios@3.3.5 typegen:install: `cd typegen && npm install`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the openapi-client-axios@3.3.5 typegen:install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?
```

**Comment by matrixise** [comment]
I don't have this issue with <= 3.3.3.

**Comment by anttiviljami** [comment]
Argh, my bad! Fixing right now.

**Comment by anttiviljami** [comment]
Fixed in openapi-client-axios@3.3.6 !
